debezium-storage/debezium-storage-rocketmq/src/main/java/io/debezium/storage/rocketmq/history/RocketMqSchemaHistory.java

```diff
@@ -0,0 +1,313 @@
+            .withType(ConfigDef.Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.ADVANCED, 0))
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDescription("The number of attempts in a row that no data are returned from RocketMQ before recover " +
+                    "completes. "
+                    + "The maximum amount of time to wait after receiving no data is (recovery.attempts) x (recovery.poll.interval.ms).")
+            .withDefault(60)
+            .withValidation(Field::isInteger);
+
+    public static final Field RECOVERY_POLL_INTERVAL_MS = Field.create(CONFIGURATION_FIELD_PREFIX_STRING
+            + "rocketmq.recovery.poll.interval.ms")
+            .withDisplayName("Poll interval during database schema history recovery (ms)")
+            .withType(ConfigDef.Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.ADVANCED, 1))
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDescription("The number of milliseconds to wait while polling for persisted data during recovery.")
+            .withDefault(1000)
+            .withValidation(Field::isLong);
+
+    private final DocumentReader reader = DocumentReader.defaultReader();
+    private String topicName;
+    private String dbHistoryName;
+    private DefaultMQProducer producer;
+    private RocketMqConfig rocketMqConfig;
+    private int maxRecoveryAttempts;
+    private Long pollInterval;
+
+    @Override
+    public void configure(Configuration config, HistoryRecordComparator comparator, SchemaHistoryListener listener, boolean useCatalogBeforeSchema) {
+        super.configure(config, comparator, listener, useCatalogBeforeSchema);
+        this.topicName = config.getString(TOPIC);
+        this.dbHistoryName = config.getString(SchemaHistory.NAME, UUID.randomUUID().toString());
+        this.maxRecoveryAttempts = config.getInteger(RECOVERY_POLL_ATTEMPTS);
+        this.pollInterval = config.getLong(RECOVERY_POLL_INTERVAL_MS);
+        LOGGER.info("Configure to store the debezium database history {} to rocketmq topic {}",
+                dbHistoryName, topicName);
+        // build config
+        this.rocketMqConfig = RocketMqConfig.newBuilder()
+                .aclEnable(config.getBoolean(RocketMqSchemaHistory.ROCKETMQ_ACL_ENABLE))
+                .accessKey(config.getString(RocketMqSchemaHistory.ROCKETMQ_ACCESS_KEY))
+                .secretKey(config.getString(RocketMqSchemaHistory.ROCKETMQ_SECRET_KEY))
+                .namesrvAddr(config.getString(RocketMqSchemaHistory.NAME_SRV_ADDR))
+                .groupId(dbHistoryName)
+                .build();
+    }
+
+    @Override
+    public void initializeStorage() {
+        super.initializeStorage();
+        LOGGER.info("try to create history topic: {}!", this.topicName);
+        TopicConfig topicConfig = new TopicConfig(this.topicName, 1, 1, 6);
```

Comment:
Instead of `6`, please use `PermName.PERM_READ | PermName.PERM_WRITE` - it's more easy to understand and also use RocketMQ predefined constant instead of hard-coded numbers.